.flake8

```diff
@@ -1,5 +1,28 @@
 [flake8]
-ignore = E203, F403, W503, FS003
+ignore =
+    # E203 and W503 are rules that conflict with Black
+    E203,
+    W503,
+
+    # FS003 weirdly thinks things are supposed to be f-strings? Seems completely broken
+    FS003,
```

Comment:
Where? This is works great on CI and locally on my machine

Reply:
Try to comment out this rule then run make flake8